Makefile

```diff
@@ -29,7 +29,7 @@ static:
 	./scripts/gobuild.sh
 
 govet:
-	go vet $(shell go list ./ecs-init/...)
+	go vet ./...
```

Comment:
why was this changed?

Reply:
this was changed because it’s not necessary and it was missing some files in ./scripts/ directory